README.md

```diff
@@ -9,7 +9,7 @@ Changes the current Ruby.
 * Updates `$PATH`.
   * Also adds RubyGems `bin/` directories to `$PATH`.
 * Correctly sets `$GEM_HOME` and `$GEM_PATH`.
-  * Users: gems are installed into `~/.gem/$ruby/$version`.
+  * Users: gems are installed into `~/.gem/$(basename /path/to/$ruby)`.
```

Comment:
`$ruby-$version` might be more understandable? Maybe even `$ruby_dir`?

Reply:
`$ruby-$version` would only be correct if the Ruby is installed with `ruby-install` or so and without changing the default installation prefix.

I would think `$ruby_dir` is an absolute path (same as `/path/to/$ruby`).
I'd be fine with `$(basename $ruby_dir)` or `$(basename /path/to/$ruby_dir)` or `$(basename $ruby_prefix)`.

`$(basename /path/to/$ruby)` is correct and I think rather clear (`/path/to/$ruby` was already used in the README)

Reply:
Wouldn't `$(basename /path/to/$ruby)` just be `$ruby`? Also, in your example `$ruby` is ambiguous. Is it the Ruby's name, name-version, arbitrary install dir name?

I would mention both `/path/to/$ruby-$version` and `~/.gem/$ruby-$version` to make the mapping explicitly clear. Not everyone who will be reading the README/ChangeLog might understand what `basename` is yet.

Reply:
I mean `arbitrary install dir name`, so maybe `/path/to/$ruby_prefix` or `/path/to/$ruby_install_dir`.
There is nothing enforcing the directories to be named `$ruby-$version` so I'd rather avoid that here.

Reply:
I tried to clarify by changing to this:

* Correctly sets `$GEM_HOME` and `$GEM_PATH`.
  * Users: gems are installed into `~/.gem/$ruby_name` (e.g., `~/.gem/ruby-2.6.3` for `~/.rubies/ruby-2.6.3`).
  * Root: gems are installed directly into `$ruby_install_dir/$gemdir` (the default).